conf/spgwu.bess

```diff
@@ -142,21 +148,33 @@ class Port:
         print('Setting up port {} on worker {}'.format(name,wid))
 
         # Detect the mode of this interface - DPDK/AF_XDP/AF_PACKET
-        mode = self.detect_mode()
+        port_mode = self.detect_mode()
```

Comment:
No need for another variable name.

if mode is None:
 mode = self.detect_mode()